src/pages/status/migration/index.jsx

```diff
@@ -200,8 +203,10 @@ function Graph(props) {
 }
 
 function Table({ details }) {
-  const [filters, setState] = useState(ORDERED
-      .reduce((filters, [status]) => ({ ...filters, [status]: true }), {}));
+  const defaultFilters = ORDERED.reduce((filters, [status]) => ({ ...filters, [status]: DEFAULT_FILTERING_STATE }), {});
+  const [filters, setState] = useState(defaultFilters);
+  filters['done'] = !DEFAULT_FILTERING_STATE;
```

Comment:
This way makes "Done" always off, I think. I can't toggle it anymore, at least. Let me add a suggestion.

Reply:
Yes. That's not what we want.